tests/IsEmptyTest.php

```
@@ -8,36 +8,28 @@
     {
         $this->assertTrue(Iterables::isEmpty([]));
         $this->assertFalse(Iterables::isEmpty(['foo']));
     }
 
-    /**
-     * @test
-     */
+    #[\PHPUnit\Framework\Attributes\Test]
     public function itReturnsTrueForEmptyArrayWithMixedKeys(): void
     {
         $this->assertTrue(Iterables::isEmpty([]));
```

PHPMD (2.15.0)

Avoid using static access to class '\Bonu\Iterable\Iterables' in method 'itReturnsTrueForEmptyArrayWithMixedKeys'. (undefined)

(StaticAccess)